### ISSUE TITLE
Change shape and gray-level models to classes

### ISSUE THREAD

**Issue by johnwmillr** [johnwmillr/ActiveShapeModels#2]
The [shape](https://github.com/johnwmillr/ActiveShapeModels/blob/master/buildShapeModel.m) and [gray-level](https://github.com/johnwmillr/ActiveShapeModels/blob/master/buildGrayLevelModel.m) models that we train should really be their own classes. That way we could right built-in visualization functions that display the models in a more convenient manner.

**Comment by kateoscar** [comment]
hello friend, i have a question here. Can this code be applied to body profile detection?

**Comment by johnwmillr** [comment]
Hi there--
Yes, you can [train your own models](https://github.com/johnwmillr/ActiveShapeModels#more-than-just-faces) on shapes other than faces. I can't guarantee the model will work well as a detector, but the code will let you use your own set of landmarks.

I hope that helps!

**Comment by kateoscar** [comment]
Thanks for your replying. it really helps me a lot. 
And i have one more question that bothers me：do training set images and test set images have to be the same size?

have a nice day~

**Comment by johnwmillr** [comment]
I'm not sure, but I don't think the dimensions of the images within the train and test sets have to be equal. What matters is that the number of landmarks across both is identical.
